tests/backup/backup_restore_basic_test.go

```diff
@@ -1318,3 +1319,197 @@ var _ = Describe("{BackupSyncBasicTest}", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			backupName = fmt.Sprintf("%s-%v", "backup", time.Now().Unix())
+			err = CreateBackupWithNamespaceLabel(backupName, SourceClusterName, backupLocationName, backupLocationUID,
+				labelSelector, orgID, clusterUid, "", "", "", "", namespaceLabel, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup [%s] creation with labels [%s]", backupName, namespaceLabel))
+		})
+		Step("Restoring backup", func() {
+			log.InfoD("Restoring backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			restoreName = fmt.Sprintf("%s-%s", restoreNamePrefix, backupName)
+			err = CreateRestore(restoreName, backupName, namespaceMapping, SourceClusterName, orgID, ctx, nil)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying restoration of backup %s", restoreName))
+			restoreNames = append(restoreNames, restoreName)
+		})
+		Step("Create schedule policy", func() {
+			log.InfoD("Creating a schedule policy")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			periodicSchedulePolicyName = fmt.Sprintf("%s-%v", "periodic", time.Now().Unix())
+			periodicSchedulePolicyUid = uuid.New()
+			periodicSchedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, 2, 5)
+			err = Inst().Backup.BackupSchedulePolicy(periodicSchedulePolicyName, periodicSchedulePolicyUid, orgID, periodicSchedulePolicyInfo)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of periodic schedule policy of interval 15 minutes named [%s]", periodicSchedulePolicyName))
+			periodicSchedulePolicyUid, err = Inst().Backup.GetSchedulePolicyUid(orgID, ctx, periodicSchedulePolicyName)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching uid of periodic schedule policy named [%s]", periodicSchedulePolicyName))
+		})
+		Step("Creating schedule backups", func() {
+			log.InfoD("Creating schedule backups")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			for _, namespace := range bkpNamespaces {
+				scheduleBackupName = fmt.Sprintf("%s-schedule-%s", BackupNamePrefix, namespace)
+				err = CreateScheduleBackupWithNamespaceLabel(scheduleBackupName, SourceClusterName, backupLocationName, backupLocationUID,
+					nil, orgID, "", "", "", "", namespaceLabel, periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of creating schedule backup with schedule name %s and labels [%v]", scheduleBackupName, namespaceLabel))
+				firstScheduleBackupName, err = GetFirstScheduleBackupName(ctx, scheduleBackupName, orgID)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup - %s", firstScheduleBackupName))
+				scheduleNames = append(scheduleNames, scheduleBackupName)
```

Comment:
Same as above. Used same code in test case "BasicAndScheduleBackupUsingNamespaceLabel"

Reply:
will take up in next PR